lib/rubycritic/generators/console_report.rb

```diff
@@ -0,0 +1,52 @@
+require "colorize"
+
+module Rubycritic
+  module Generator
+    class ConsoleReport
+      def initialize(analysed_modules)
+        @analysed_modules = analysed_modules
+      end
+
+      def generate_report
+        reports = @analysed_modules.sort.map do |analysed_module|
+          generate_module_report analysed_module
+        end
+        puts reports.join("\n")
+      end
+
+      private
+
+      def generate_module_report(mod)
+        report = <<-EOF
+#{mod.name.send(mod.rating.color)}:
```

Comment:
[Law of Demeter](http://devblog.avdi.org/2011/07/05/demeter-its-not-just-a-good-idea-its-the-law/) violation